addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -818,6 +818,18 @@ const Wysiwyg = Widget.extend({
             observerData.observer.disconnect();
         }
     },
+    /**
+     * Activates the link tool for the image.
+     */
+    toggleLinkOnImage() {
+        const buttonEl = document.querySelectorAll('we-button[data-set-link]')[1];
+        if (!buttonEl.classList.contains('active')) {
+            buttonEl.click();
+        }
+        setTimeout(function () { // Wait for the URL input to appear.
+            document.querySelectorAll('we-input[data-set-url]')[1].querySelector('input').focus();
```

Comment:
I know it's not ideal to search for an item from the document but I haven't found any other way to access the editor's items from this file.